diff --git a/autoload/nerdtree.vim b/autoload/nerdtree.vim
index 539e783..05725c8 100644
--- nerdtree/autoload/nerdtree.vim
+++ nerdtree/autoload/nerdtree.vim
@@ -135,7 +135,7 @@ function! nerdtree#closeTree()
 
         call nerdtree#exec(nerdtree#getTreeWinNum() . " wincmd w")
         close
-        call nerdtree#exec(bufwinnr(bufnr) . " wincmd w")
+        execute bufwinnr(bufnr) . " wincmd w"
     else
         close
     endif
